data_steward/cdr_cleaner/cleaning_rules/repopulate_person_post_deid.py

```diff
@@ -160,12 +165,15 @@
   gender_source_value,
   gender_source_concept_id,
   CASE
-    WHEN race_concept_id = 0 THEN "None Indicated"
+    WHEN race_concept_id = 0 THEN "{{aou_custom_value}}"
   ELSE
   race_source_value
-END
-  AS race_source_value,
-  race_source_concept_id,
+END AS race_source_value,
+  CASE
+    WHEN race_source_concept_id = 0 THEN {{aou_custom_concept}}
+    ELSE
+    race_source_concept_id
+END AS race_source_concept_id,
   ethnicity_concept.concept_code ethnicity_source_value,
```

Comment:
This change is part of the acceptance criteria.  Additionally and unknowingly, the replacement at line 168 is part of DC-2827.  @nishanthpp93 mentioned it might make more sense to wrap both these tickets into one PR since they are so closely related.